docs/typescript.md

```diff
@@ -1,4 +1,15 @@
-# Usage
+# TypeScript Client
+
+<br>
```

Comment:
Can you both align these vertically and use the badges from the Github that auto pull the version and such? You should be able to simply copy from https://github.com/dydxprotocol/solo/blob/master/README.md